espnet2/bin/enh_scoring.py

```diff
@@ -64,6 +66,41 @@ def scoring(
         format="%(asctime)s (%(module)s:%(lineno)d) %(levelname)s: %(message)s",
     )
 
+    if dnsmos:
+        if dnsmos_args["mode"] == "local":
+            from espnet2.enh.layers.dnsmos import DNSMOS_local
+
+            if not Path(dnsmos_args["primary_model"]).exists():
+                raise ValueError(
+                    f"The primary model '{dnsmos_args['primary_model']}' does not exist. "
+                    "You can download the model from https://github.com/microsoft/"
+                    "DNS-Challenge/tree/master/DNSMOS/DNSMOS"
+                )
+            if not Path(dnsmos_args["p808_model"]).exists():
```

Comment:
A minor suggestion, this dnsmos is very specific to p808 model. It would be better if this can be more flexible for future.

Reply:
I don't know if there are other options for the DNS model. I just followed the script provided by the official repository https://github.com/microsoft/DNS-Challenge/blob/master/DNSMOS/dnsmos_local.py#LL23C47-L23C47